classes/class-wc-connect-stripe.php

```diff
@@ -156,5 +162,33 @@ private function get_default_stripe_config() {
 
 			return $result;
 		}
+
+		public function maybe_show_banner() {
+			if ( 'yes' !== WC_Connect_Options::get_option( 'banner_stripe', null ) ) {
+				return;
+			}
+
+			if ( isset( $_GET[ 'wcs_stripe_code' ] ) ) {
```

Comment:
Should there be an `isset()` check for `wcs_stripe_state` as well?

Reply:
Done in 6b4bed8